docs/docs/reference/policies.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Policies
+---
+
+Starting with Chainloop [0.93.8](https://github.com/chainloop-dev/chainloop/releases/tag/v0.93.8), operators can attach policies to contracts. 
+These policies will be evaluated against all attestation metadata and materials in any workflow with that contract.
+
+### Policy specification
+A policy can be defined in a YAML document, like this:
+```yaml
+# sbom-licenses.yaml
+apiVersion: workflowcontract.chainloop.dev/v1
+kind: Policy
+metadata:
+  name: sbom-licenses # (1)
+spec:
+  type: SBOM_CYCLONEDX_JSON # (2)
+  embedded: | # (3)
+    package main
+
+    deny[msg] {
+      count(without_license) > 0
+      msg := "SBOM has components without licenses"
+    }
+
+    without_license = {comp.purl |
+      some i
+      comp := input.components[i]
+      not comp.licenses
+    }
+```
+In this particular example, we see:
+* (1) policies have a name
+* (2) they can be optionally applied to a specific type of material (check [the documentation](./operator/contract#material-schema) for the supported types). If no type is specified, a material name will need to be provided explicitly in the contract.
+* (3) they have a policy script that it's evaluated against the material (in this case a CycloneDX SBOM report). Currently, only [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/#learning-rego) policies are supported.
+
+Policy scripts could also be specified in a detached form:
+```yaml
+...
+spec:
+  type: SBOM_CYCLONEDX_JSON
+  path: my-script.rego
+```
+
+### Applying policies to contracts
+When defining a contract, a new `policies` section can be specified. Policies can be applied to any material, but also to the attestation statement as a whole.
+```yaml
+schemaVersion: v1
+materials:
+  - name: sbom
+    type: SBOM_CYCLONEDX_JSON
+  - name: another-sbom
+    type: SBOM_CYCLONEDX_JSON
+policies:
+  materials: # policies applied to materials
+    - ref: sbom-licenses.yaml # (1)
```

Comment:
It's not clear if this is a local file that needs to be somewhere or a name? Did we end up adding file://... to the ref?

Reply:
It's using the `blob.LoadFileOrURL` method from cosign, which defaults to the local filesystem if no `http://`, `https://` or `env://` is provided.

Reply:
I'll make sure it's clear in the docs.